tsconfig.json

```diff
@@ -1,17 +1,13 @@
 {
   "compilerOptions": {
-    "declaration": true,
     "esModuleInterop": true,
-    "inlineSources": true,
     "lib": ["ES2020"],
     "module": "CommonJS",
-    "moduleResolution": "Node",
-    "outDir": "dist",
-    "rootDir": "src",
-    "sourceMap": true,
+    "moduleResolution": "node",
+    "noEmit": true,
+    "noErrorTruncation": true,
     "strictNullChecks": true,
-    "target": "ES2017",
-    "typeRoots": ["./node_modules/@types"]
+    "target": "es2017"
```

Comment:
Worth noting that in the module template we use a target of `es2020`. I've added a ticket here: https://github.com/MetaMask/eth-sig-util/issues/353